xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -355,14 +355,17 @@
     #if ("$!dateFormat" == '')
       #set ($dateFormat = $xwiki.getXWikiPreference('dateformat', 'yyyy/MM/dd HH:mm'))
     #end
+    #set($dateRangeFormat = $dateFormat + " - " + $dateFormat)
+    <div id="xwiki-livetable-${htmlLiveTableId}-filter-${foreach.count}-description" class='sr-only'>
```

Comment:
I'm wondering if this shouldn't rather be hidden to avoid that screen readers will read this outside the context of the input. The way it currently is, I assume a screen reader will read this before the input and thus out of context, where this description might be confusing.

Reply:
```suggestion
    <div id="xwiki-livetable-${htmlLiveTableId}-filter-${foreach.count}-description" hidden>
```

Reply:
Seems okay, `display: none` hides everything, so I wanted to be sure.

[Video demo with this change](https://up1.xwikisas.com/#GgLbHbyLlBPRSbAgV5F0og)